azure-pipelines-dependency-graph-testing.yml

```diff
@@ -148,10 +148,12 @@ stages:
 
           - script: |
               pip install pytest pytest-cov pytest-azurepipelines
-              test_files=$(python scripts/determine_tests_to_run.py --depth 3 --ignore 'tests/cli' 'tests/integration/usage_statistics')
-              if [ $? -eq 0 ]; then
-                echo $test_files | xargs pytest $(GE_pytest_opts) --napoleon-docstrings --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html
+              echo "Tests identified by dependency graph test runner [dgtest]:"
+              python scripts/determine_tests_to_run.py --depth 3 --ignore 'tests/cli' 'tests/integration/usage_statistics' | tee .dgtest
+              if [ -s .dgtest ]; then
+                cat .dgtest | xargs pytest $(GE_pytest_opts) --napoleon-docstrings --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html
```

Comment:
Using `tee` to print script results to both the terminal and a file (this helps with debugging since we have the output in the logs).